phpBB/config/default/container/services_attachment.yml

```diff
@@ -0,0 +1,31 @@
+services:
+    attachment.delete:
+        class: phpbb\attachment\delete
+        scope: prototype
+        arguments:
+            - @config
+            - @dbal.conn
+            - @filesystem
+            - @attachment.resync
+            - %core.root_path%
+
+    attachment.resync:
+        class: phpbb\attachment\resync
+        scope: prototype
+        arguments:
+            - @dbal.conn
+
+    attachment.upload:
```

Comment:
these services seems to be 3 different part of an attachments manager.

I think i'd prefer to have and expose a global manager (using these services).